Assets/Scripts/DayNightCycle/SyncTimeButtonHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Assets.Scripts.Services;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Assets.Scripts.DayNightCycle
+{
+    class SyncTimeButtonHandler : MonoBehaviour
```

Comment:
Is this class even used? I can't seem to find any references in the scenes I tested it with. (City, Outside the Walls)

Reply:
It used to be the syncbutton script, will remove it.